ci/builders/mac_host_engine.json

```diff
@@ -241,19 +241,6 @@
                     "sdk_version": "15a240d"
                 }
             },
-            "tests": [
```

Comment:
Ohhh, shoot. My bad. These tests are passing on the release branch. It's the `impeller-golden` tests down on line 470 that are failing, and which should be removed on the release branch.

Reply:
Updated